### PR TITLE
Io turtlesim qos

### PR DESCRIPTION
**Pull request by abolinsky**
Enforce correct dependency topology for package builds.
Cut out a few actions from manifest.

### CODE REVIEW

File: io_turtlesim/manifests/command_center.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
             "topics": [
               {
                 "name": "/cmd_vel",
-                "qos": "low"
+                "qos": "max"
```
Comment (hendrik-skubch) [inline io_turtlesim/manifests/command_center.json]: Erm, no just low please

File: io_turtlesim/manifests/command_center.json [hunk 2]
```diff
@@ -88,24 +88,6 @@
...
-            "name": "/turtle_4/goto_action"
-          },
-          {
-            "name": "/turtle_3/goto_action"
-          },
-          {
-            "name": "/turtle_2/goto_action"
-          },
           {
             "name": "/turtle_1/goto_action"
```
Comment (hendrik-skubch) [inline io_turtlesim/manifests/command_center.json]: The tutorial mentions at least actions for the first three turtles.

File: io_turtlesim/manifests/turtle.json [hunk 3]
```diff
@@ -28,31 +28,13 @@
               },
               {
                 "name": "/sim/cmd_vel",
-                "qos": "low"
+                "qos": "max"
```
Comment (hendrik-skubch) [inline io_turtlesim/manifests/turtle.json]: Low, please

**Review by hendrik-skubch — CHANGES_REQUESTED** [review]

File: io_turtlesim/manifests/command_center.json (hunk 1 above)
Comment (gauthamm) [reply io_turtlesim/manifests/command_center.json]: If we are changing it to low, we need to update the turtlesim tutorial document too.

**Comment by abolinsky** [comment]
Gautham already merged the bulk of this into master

**Pull request closed without merging**